### ISSUE TITLE
App keep stopping

### ISSUE THREAD

**Issue by mishra0007** [Dheeraj22/Professor_DK#9]
Whenever I hit the play button, the app keeps stopping. Even though all are compiled and just view wearing in the main Activity

**Comment by ca-blan** [comment]
Yes, I have the same problem. I have even changes the device and s¡the app still stopping. 
Does anyone know why? pleaseee!

**Comment by Saibala89** [comment]
Hi,

Just follow the simple steps to resolve the mentioned issue.

1. Goto Run > Edit configuration > Launcher need to be as **Default Activity**.
2. After that Go to File > select Invalidate caches / Restart > proceed.

After following this two steps sure it will work.

**Comment by adityakishan2018** [comment]
Exactly same issue!!
